test/functional/settings-e2e-specs.js

```diff
@@ -0,0 +1,43 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import _ from 'lodash';
+import AndroidDriver from '../..';
+import DEFAULT_CAPS from './desired';
+import { sleep } from 'asyncbox';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+let defaultCaps = _.defaults({
+  androidInstallTimeout: 90000,
+  browserName: 'chrome'
+}, DEFAULT_CAPS);
+
+describe.skip('toggle wifi tests', () => {
```

Comment:
Generally speaking with set the `REAL_DEVICE` environment variable when running on real devices. So we skip like https://github.com/appium/appium-android-driver/blob/master/test/functional/webview-browser-tester-e2e-specs.js#L26

you can also skip based on `process.env.TRAVIS` to not run in CI.